command/agent/auth/auth.go

```diff
@@ -126,7 +128,7 @@ func (ah *AuthHandler) Run(ctx context.Context, am AuthMethod) error {
 		}
 
 		// Create a fresh backoff value
-		backoff := 2*time.Second + time.Duration(ah.random.Int63()%int64(time.Second*2)-int64(time.Second))
+		backoff := eb.NextBackOff()
```

Comment:
Looks like this call can return `backoff.Stop` (-1), though maybe only if we don't configure `DefaultMaxElapsedTime`?

Reply:
I'd assume that `NewExponentialBackOff` uses the `DefaultMaxElapsedTime` value unless otherwise specified, which means that this would return -1 after 15 minutes.